server/Dockerfile

```diff
@@ -1,24 +1,30 @@
-FROM node:16-buster AS BUILD_IMAGE
-LABEL org.opencontainers.image.source https://github.com/ccmbioinfo/osmp
-
-
-WORKDIR /usr/app
-
-RUN wget -O /home/node/cadd_wgs_ghr38_index.gz.tbi  https://krishna.gs.washington.edu/download/CADD/v1.6/GRCh38/whole_genome_SNVs_inclAnno.tsv.gz.tbi \ 
-    && chown node:node /home/node/cadd_wgs_ghr38_index.gz.tbi
-
-COPY . . 
-
-RUN yarn && yarn run build
-
-FROM node:16-slim as DEPLOY_IMAGE 
-
-WORKDIR /usr/app
-
-COPY --from=BUILD_IMAGE /home/node/cadd_wgs_ghr38_index.gz.tbi /home/node/cadd_wgs_ghr38_index.gz.tbi
-COPY --from=BUILD_IMAGE /usr/app/dist ./dist
-COPY --from=BUILD_IMAGE /usr/app/node_modules ./node_modules
-COPY --from=BUILD_IMAGE /usr/app/package.json ./package.json
+# This stage will probably be removed when we index GRCh38 ourselves
+FROM curlimages/curl AS tbi
+WORKDIR /var/tmp
+RUN curl --output cadd_wgs_ghr38_index.gz.tbi https://krishna.gs.washington.edu/download/CADD/v1.6/GRCh38/whole_genome_SNVs_inclAnno.tsv.gz.tbi
```

Comment:
We can remove the `tbi` step altogether and paste `https://krishna.gs.washington.edu/download/CADD/v1.6/GRCh38/whole_genome_SNVs_inclAnno.tsv.gz.tbi` as the index file in `fetchCaddAnnotations`, in a similar fashion of how the remote 37 file is passed in.